.dumi/theme/builtins/Previewer/CodeBlockButton.tsx

```
@@ -57,24 +25,61 @@ const CodeBlockButton: React.FC<CodeBlockButtonProps> = ({ title, dependencies =
+      openHituCodeBlockFn();
+      return;
+    }
+    setLoading(true);
+    const script = document.createElement('script');
+    script.src =
+      'https://renderoffice.a' +
+      'lipay' +
+      'objects.com/p' +
+      '/yuyan/180020010001206410/parseFileData-v1.0.1.js' +
+      `?t=${Date.now()}`;
+    script.async = true;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)